apps/test/util/testUtils.js

```diff
@@ -118,17 +118,39 @@ exports.runOnStudioTick = function (tick, fn) {
 };
 
 /**
+ * In a bunch of our tests we wrap app.onTick. When we do so, we now also store
+ * a reset function that unwraps it. This method calls all of those functions
+ * at the end of a test to make sure that we're unwrapping everything before
+ * the next test.
+ */
+var wrappersToReset = [];
+exports.resetAppTicks = function () {
+  wrappersToReset.forEach(function (reset) {
+    reset();
+  });
```

Comment:
I can't find a test that wraps `onTick` twice; it might be easier just to enforce that you can't wrap it more than once, than to make the wrapping and unwrapping order-independent.